app/templates/tasklist.html

```diff
@@ -11,58 +12,44 @@
     <h1 class="govuk-heading-xl">
       {{application_meta_data["fund_name"]|kebab_case_to_human}} application
     </h1>
-    {% if not application_response["status"] == application_meta_data["submitted_status"] %}
 
-    <p class="govuk-body-l govuk-!-font-weight-bold">
+    {% if application["status"] == application_meta_data["completed_status"] %}
+    {{tasklist_submit(application_id, application_meta_data, form, url_for('routes.submit_application'))}}
+    {% elif not application["status"] == application_meta_data["submitted_status"] %}
+    <h2 class="govuk-body-l govuk-!-font-weight-bold">
       Status:
-      {% set completed_count = application_meta_data["number_of_completed_sections"] %}
-      {% set section_count = application_meta_data["number_of_sections"] %}
-      {% if completed_count == 0 %}
+      {% set completed_count = application_meta_data["number_of_completed_forms"] %}
+      {% set in_progress_count = application_meta_data["number_of_in_progress_forms"] %}
+      {% set form_count = application_meta_data["number_of_forms"] %}
```

Comment:
Are these used?

Reply:
yes they are used to display counts in tasklist.html